doc/user_guide/tools.rst

```diff
@@ -58,12 +58,12 @@ examples, the data is stored in a numpy array in the
 :py:attr:`~.signal.BaseSignal.original_metadata` attribute, the mapped parameters
 in the :py:attr:`~.signal.BaseSignal.metadata` attribute and the axes
 information (including calibration) can be accessed (and modified) in the
-:py:attr:`~.signal.BaseSignal.axes_manager` attribute.
+:py:class:`~.axes.:py:class:`~.axes.AxesManager`` attribute.
 
 Signal initialization
 ---------------------
 
-Many of the values in the :py:attr:`~.signal.BaseSignal.axes_manager` can be
+Many of the values in the :py:class:`~.axes.:py:class:`~.axes.AxesManager`` can be
```

Comment:
Actally, maybe I somehow changed that last line myself? It seems I did. I'll do it again Sorry.